rules/do-you-know-when-to-mockup-your-api/rule.md

```diff
@@ -1,27 +1,132 @@
 ---
 type: rule
 archivedreason: 
-title: Comments - Do you know when to mockup your APIs?
+title: Do you know when to mockup your API?
 guid: eee9e703-e6bd-40f8-9f65-8a12a3d92f8d
 uri: do-you-know-when-to-mockup-your-apis
 created: 2021-05-11T22:00:01.0000000Z
 authors:
+- title: William Liebenberg
+  url: https://ssw.com.au/people/william-liebenberg
 - title: Bryden Oliver
   url: https://ssw.com.au/people/bryden-oliver
 related: 
 - do-you-know-how-to-brand-your-api-portal
 - do-you-know-the-best-tools-to-manage-apis
 redirects:
-- comments-do-you-know-when-to-mockup-your-apis
+- do-you-know-when-to-mockup-your-apis
```

Comment:
The redirects list should have all the rule's old URI's.
Please change it back to 
```yml
redirects:
- comments-do-you-know-when-to-mockup-your-apis
```

Reply:
done